packages/core-js-compat/package.json

```diff
@@ -9,10 +9,10 @@
   "main": "index.js",
   "dependencies": {
     "browserslist": "^4.6.3",
-    "core-js-pure": "3.1.4",
     "semver": "^6.1.3"
   },
   "devDependencies": {
+    "core-js-pure": "3.1.4",
```

Comment:
This is still used by `packages/core-js-compat/src/build-entries.js` (which isn't included in the published package, so it can be a dev dependency).

Reply:
Since it's a similar case like in `index`, it would be better also replace it.